datadog/dogstatsd/base.py

```diff
@@ -512,13 +584,20 @@ def _send_to_server(self, packet):
                 self.bytes_dropped += len(telemetry)
                 self.packets_dropped += 1
 
-    def _xmit_packet(self, packet, telemetry):
+    def _xmit_packet(self, packet, account, is_telemetry):
```

Comment:
I don't get what `account` is and what ` self._xmit_packet(packet, self._telemetry, False)` means then

Reply:
So, this what each means: 
- `account` is to tell `_xmit_packet()` if we want to do accounting of the packet getting sent.
- `telemetry` is to tell `_xmit_packet()` if we're sending an a telemetry packet or not. 

But I see what you're saying, we don't need two parameters since we'll never want to `account` for telemetry packets. OK, I'll fix this.